analysers/analyser_merge_power_branch_FR.py

```diff
@@ -47,14 +47,14 @@ def __init__(self, config, logger = None):
             Conflate(
                 select = Select(
                     types = ["nodes"],
-                    tags = [
-                        {"power": ["tower", "pole", "portal", "insulator", "connection"], "operator": "RTE", "line_management": {"regex": r"(^|\(|\|)branch(\||\)|$)"}}]),
+                    tags = [{"power": ["tower", "pole", "portal", "insulator", "connection"], "operator": "RTE", "line_management": {"regex": r"(^|\(|\|)branch(\||\)|$)"}}]),
                 osmRef = "ref:FR:RTE",
                 conflationDistance = 200,
                 mapping = Mapping(
                     static1 = {
                         "power": "tower",
-                        "operator": "RTE"},
+                        "operator": "RTE",
+                        "operator:wikidata": "Q2178795"},
```

Comment:
Same here about new tag in static1.

Reply:
There are less than 2k towers covered by this analysis, and only 124 are missing operator:wikidata.
It sounds fine to let it in static1

Reply:
OK. Thank.